packages/ai/src/ai/modules/remote/__init__.py

```
@@ -6,7 +6,7 @@
 from .remote_pipe import remote_pipe
 
 
-def initModule(server: WebServer, config: Autopipe = {}):
+def initModule(server: WebServer, config: Autopipe = None):
```

Ruff (0.15.7)

[warning] Unused function argument: `config`

(ARG001)